web/views/setting_view.ex

```diff
@@ -1,8 +1,17 @@
 defmodule CoursePlanner.SettingView do
   @moduledoc false
   use CoursePlanner.Web, :view
+  alias CoursePlanner.SharedView
 
-  def title do
-    "Settings"
+  def setting_input(form, field, type, label) do
+    case type do
```

Comment:
hat do you think of removing the cases which have the same behavior as the default? for example
```elixir
"text"    -> SharedView.form_textarea(form, field, label: label)
"boolean" -> SharedView.form_select(form, field, ["True", "False"], label: label)
_         -> SharedView.form_text(form, field, label: label)
```